src/main/java/frc/robot/subsystems/yeeter/YeeterIO.kt

```diff
@@ -10,26 +10,26 @@ import frc.robot.CANDevice
 import org.littletonrobotics.junction.LogTable
 import org.littletonrobotics.junction.inputs.LoggableInputs
 
-interface ShooterIO {
-    class ShooterRawInputs : LoggableInputs {
+interface YeeterIO {
+    class YeeterRawInputs : LoggableInputs {
 
-        var mainVelocity = 0.0
-        var secondaryVelocity = 0.0
+        var yeetMainVelocity = 0.0
```

Comment:
Why add the `yeet` prefix? It seems already clear since these are the `Yeeter` inputs.